setup.py

```diff
@@ -710,14 +710,25 @@ def add_cross_compiling_paths(self):
                         elif line.startswith("End of search list"):
                             in_incdirs = False
                         elif (is_gcc or is_clang) and line.startswith("LIBRARY_PATH"):
-                            for d in line.strip().split("=")[1].split(":"):
+                            for d in line.strip().split("=")[1].split(os.pathsep):
                                 d = os.path.normpath(d)
-                                if '/gcc/' not in d:
+                                if '/gcc/' not in d and '/clang/' not in d:
                                     add_dir_to_list(self.compiler.library_dirs,
                                                     d)
                         elif (is_gcc or is_clang) and in_incdirs and '/gcc/' not in line and '/clang/' not in line:
                             add_dir_to_list(self.compiler.include_dirs,
                                             line.strip())
+            if is_clang:
+                ret = run_command('%s -print-search-dirs >%s' % (cc, tmpfile))
```

Comment:
Suggestion: Use format strings instead?
```suggestion
                ret = run_command(f'{cc} -print-search-dirs > {tmpfile}')
```

Reply:
This was pretty much copy/pasted from the loop above, followed how it was doing things.